test/token/ERC20/extensions/ERC20Votes.test.js

```diff
@@ -62,209 +41,12 @@ contract('ERC20Votes', function (accounts) {
       await this.token.mint(holder, 1);
     }
     const block = await web3.eth.getBlockNumber();
-    expect(await this.token.numCheckpoints(holder)).to.be.bignumber.equal('6');
     // recent
     expect(await this.token.getPastVotes(holder, block - 1)).to.be.bignumber.equal('5');
     // non-recent
     expect(await this.token.getPastVotes(holder, block - 6)).to.be.bignumber.equal('0');
   });
 
-  describe('set delegation', function () {
```

Comment:
Let's add back all of the tests for ERC20Votes, so we can make sure that this PR is purely a refactor and doesn't change behavior in unexpected ways!

We should remove the redundant tests in a subsequent PR.